ngx.yml

```diff
@@ -0,0 +1,13 @@
+---
+profile:  qcom_ap-dk01.1-c1
```

Comment:
this is dk01, hence the file has to be called qcom_ap-dk01.1-cs1.yml. I am not sure why we have a build profile for a reference kit.